helloworld-mutual-ssl-secured/pom.xml

```diff
@@ -44,8 +44,12 @@
     </licenses>
 
     <properties>
-        <!-- The versions for BOMs, Dependencies and Plugins -->
-        <version.server.bom>30.0.0.Final</version.server.bom>
+         <!-- Version for the server -->
```

Comment:
I think <!-- Version for the server --> isn't aligned with the rest.

Reply:
fixed!